test/QuestionAPI-test.js

```diff
@@ -294,13 +290,13 @@ describe("Question API Contract", function () {
       const questionIDtx7 = await questionAPI.connect(xmetricaddr1).createQuestion("ipfs://last", 1);
       await questionIDtx7.wait();
 
-      const latestQuestion = await questionAPI.currentQuestionId();
+      const latestQuestion = await bountyQuestion.getMostRecentQuestion();
 
       const allquestionsByState = await questionStateController.getQuestionsByState(new BN(questionState.VOTING), latestQuestion, new BN(1000));
-      expect(allquestionsByState.length).to.equal(8);
+      expect(allquestionsByState.length - 1).to.equal(8);
```

Comment:
If you removed a question shouldn't .length equal 7?
I tried to name tests so you see how many questions were created